grpc/src/test/proto/testing/grpc/transcoding.proto

```diff
@@ -367,3 +380,13 @@ message EchoNestedMessageRequest {
 message EchoNestedMessageResponse {
   TopLevelMessage.NestedMessage nested = 1;
 }
+
+message ArbitraryHttpWrappedRequest {
+  string request_id = 1;
+  google.api.HttpBody body = 2;
+}
+
+message ArbitraryHttpWrappedResponse {
+  string response_id = 1;
+  google.api.HttpBody body = 2;
+}
```

Comment:
I am not 100% sure how transcoder should behave when http body is a sub field. There is also `extensions` field which I did not understand either.

Reply:
I think it's fine until we hear more feedback on this feature from early adoptors. It won't harm at all anyway if a user is not using this feature, so I'd be happy to merge it even if sub-field and extensions cases are unsupported in this PR. :smile:

Reply:
Sure, let me come back to this PR. I think I have left some gaps therefore this PR was forgotten in the Draft stage.

Reply:
Do you know if there is anyone waiting for this feature to be released? That would be great to get some feedbcak.

Reply:
None that I'm aware of, but I know the colleagues of @hyangtack use HTTP/JSON transcoding, so they might be interested. If we release this and mention it in our release note, someone might start tinkering with it though! :wink:

Reply:
I have updated the code, PTAL at the test cases and LMK what other tests we can run. For now those two seem like fine to me.